content/applications/websites/blog.rst

```diff
@@ -0,0 +1,117 @@
+====
+Blog
+====
+
+**Odoo Blog** is a content management system that allows to create and manage blog posts on your
+website to engage your audience and build a community.
+
+Creating a blog
+===============
+
+To create or edit a blog, go to :menuselection:`Website --> Configuration --> Blogs: Blogs`. Click
+:guilabel:`New` and fill in the required information: add a :guilabel:`Blog Name`, a :guilabel:`Blog
+Subtitle`, and if needed, select the :guilabel:`Website` you want to restrict the blog to.
+
+To access your blogs, go to your :guilabel:`website` and click on :guilabel:`Blog` on the top menu.
+Your blogs appear in the sub-menu bar.
+
+Writing a blog post
+===================
+
+Go to your website and click :guilabel:`+New` in the top-right corner. Select :guilabel:`Blog Post`,
+:guilabel:`Select blog`, and add a blog post :guilabel:`Title`.
+
+Write your blog post and customize your page’s content using the website builder.
+
+.. tip::
+   - Illustrate your articles with copyright-free images from :doc:`Unsplash <website/configuration/unsplash>`.
+   - Type `/` in the text editor to format and craft your text.
+
+.. important::
+   Toggle the switch in the upper-right corner from :guilabel:`Unpublished` to :guilabel:`Published`
+   to make your article visible.
+
+Using tags
+==========
+
+Tags help you categorize and organize your blog posts and offer a user-friendly way to facilitate
+search.
+
+To create a tag, go to :menuselection:`Website --> Configuration --> Blogs: Tags` and click
+:guilabel:`New`. You can also open an existing tag to edit it. Fill in the required information:
+
+- :guilabel:`Name`: your tag’s name;
+- :guilabel:`Category`: the category your tag belongs to. You can select an existing one, create a
+  new one, or leave it empty;
+- :guilabel:`Used in`: to apply your tag to existing blog posts, click :guilabel:`Add a line`,
+  select your posts, and click :guilabel:`Select`. Click :guilabel:`New` if your tag applies to a
+  post you want to create on the fly.
+
+.. note::
+   To show tags on your Blog dashboard page, click :menuselection:`Edit --> Customize`, and toggle
+   the switch of :guilabel:`Sidebar` and then of :guilabel:`Tags List`.
+
+Customizing content
+===================
+
+You can enable the features related to your blog dashboard or a specific blog post according to your
+needs.
+
+Customizing your blog dashboard
+-------------------------------
+
+From your Blog dashboard page, click :menuselection:`Edit --> Customize`, and toggle the switch of
+the following options to activate them:
+
+:guilabel:`Top Banner`: select :guilabel:`Name/Latest Post` to display the name of the latest post
```

Comment:
I know the option is called "Name/Latest Post" but I'd suggest to clarify in the description "to display the TITLE of the latest post", "name" is just weird. But it's nitpicking